identity-account/src/account/account.rs

```diff
@@ -466,6 +524,12 @@ impl Config {
     self
   }
 
+  /// Sets the account auto-publish behaviour.
+  pub fn autopublish(mut self, value: bool) -> Self {
```

Comment:
Nit: this comment should also explain what auto-publish does and the default value? This probably applies to all config options really.

E.g.
```
  /// Sets the account auto-publish behaviour.
  /// - `true` => publish to the Tangle on every change
  /// - `false` => never publish automatically
  /// 
  /// Default: `true`
  pub fn autopublish(mut self, value: bool) -> Self {
```